app/android/app/src/main/java/com/proofofpassportapp/MainActivity.kt

```
@@ -29,12 +30,22 @@ class MainActivity : ReactActivity() {
   override fun onNewIntent(intent: Intent) {
     super.onNewIntent(intent)
     Log.d("MAIN_ACTIVITY", "onNewIntent: " + intent.action)
-    RNPassportReaderModule.getInstance().receiveIntent(intent)
+    try {
+      RNPassportReaderModule.getInstance().receiveIntent(intent)
+    } catch (e: IllegalStateException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)